ui/lib/kv/addon/components/page/secret/details.hbs

```diff
@@ -64,11 +64,11 @@
     <li>
       <LinkTo @route="secret.paths" @models={{array @secret.backend @path}} data-test-secrets-tab="Paths">Paths</LinkTo>
     </li>
-    {{#if @secret.canReadMetadata}}
+    {{#if @canReadMetadata}}
```

Comment:
Changed this because I noticed while testing control groups that this tab would show sometimes because `@secret` model didn't exist